### ISSUE TITLE
[ERROR] Unsupported numpy data type: i64. Cannot implicitly convert to tensorrt.Weights.

### ISSUE THREAD

**Issue by zzlnys** [NVIDIA-AI-IOT/torch2trt#870]
[06/17/2023-20:06:23] [TRT] [I] [MemUsageChange] Init CUDA: CPU +522, GPU +0, now: CPU 10870, GPU 2466 (MiB)
[06/17/2023-20:06:23] [TRT] [I] [MemUsageSnapshot] Begin constructing builder kernel library: CPU 10923 MiB, GPU 2466 MiB
[06/17/2023-20:06:23] [TRT] [I] [MemUsageSnapshot] End constructing builder kernel library: CPU 11103 MiB, GPU 2510 MiB
Warning: Encountered known unsupported method torch.empty
Warning: Encountered known unsupported method torch.is_grad_enabled
Warning: Encountered known unsupported method torch.is_grad_enabled
Warning: Encountered known unsupported method torch.Tensor.uniform_
Warning: Encountered known unsupported method torch.is_grad_enabled
Warning: Encountered known unsupported method torch.empty
Warning: Encountered known unsupported method torch.empty
Warning: Encountered known unsupported method torch.zeros
Warning: Encountered known unsupported method torch.ones
[ERROR] Unsupported numpy data type: i64. Cannot implicitly convert to tensorrt.Weights.

How should I fix the error reported above after using TensorRT?

**Comment by itachi1232gg** [comment]
try convert you model to eval mode by `model.eval()`, hope it can help
